modules/admin_manual/pages/troubleshooting/providing_logs_and_config_files.adoc

```diff
@@ -0,0 +1,34 @@
+= Providing log files and configurations
+
+When you report a problem to ownCloud Support or our Forum (central) you will be asked to provide certain log files or configs
+for our engineers or other users to better understand your issue, your specific configuration and the cause of the problem.
+Here are instructions how to collect those files.
+
+'''
+
+== Config Report
+You can use the webUI or the command line to generate a config report.
+Please note that you have to have the configreport app enabled. You can enable this app with:
+`occ market:install configreport`  or  `occ:app enable configreport`
+
+=== web UI
+Navigate to: admin > settings > general > "Generate Config report" > "Download ownCloud config report" 
```

Comment:
For buttons use https://github.com/owncloud/docs/blob/master/docs/best-practices.md#keyboard-shortcuts-and-ui-button-text